toolkit/templates/user-research-consent-banner.html

```diff
@@ -0,0 +1,11 @@
+<div class="user-research-banner">
```

Comment:
We could instead use `id` as opposed to `class` as we should only be having one of these. The GOV.UK cookie banner seems to use `id="global-cookie-message`. I don't feel too strongly on this though. Happy to go with what you feel is best.

Reply:
I don't feel too strongly but I tend to use classes where ever possible and only IDs when actually required. Conflicting IDs can sometimes break older browsers